cuML/src/random_projection/rproj.hxx

```diff
@@ -0,0 +1,218 @@
+/*
+ * Copyright (c) 2018, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <vector>
+#include <unordered_set>
+#include <random>
+
+#include "rproj_c.h"
+#include "utils.hxx"
+#include <linalg/cublas_wrappers.h>
+#include <linalg/cusparse_wrappers.h>
+#include <cuda_utils.h>
+#include <common/cumlHandle.hpp>
+
+namespace ML {
+
+	using namespace MLCommon;
+	using namespace MLCommon::LinAlg;
+
+	/**
+	 * @brief generates a gaussian random matrix
+	 * @output param random_matrix: the random matrix to be allocated and generated
+	 * @input param params: data structure that includes all the parameters of the model
+	 * @input param stream: cuda stream
+	 */
+	template<typename math_t>
+	void gaussian_random_matrix(rand_mat<math_t> *random_matrix, paramsRPROJ& params,
```

Comment:
isn't there more to this function definition? I feeel like a line might've gotten deleted, or is the `,` supposed to be `)`

Reply:
It does not appear the same on my editor, there's simply too much indentation. If you look at the raw file it ends with "cudaStream_t stream)". Will fix this.